### DISCUSSION TITLE
Train loss and test loss error

### DISCUSSION THREAD

**Discussion by VectorScience** [lululxvi/deepxde#1227] [Q&A]
Hey guys, I found my output like this:
![image](https://user-images.githubusercontent.com/87429104/229071790-21953a60-b358-4a79-b807-6f202d8f5b7e.png)

1. Why there is test loss beside train loss? And they are the same.
2. Why I have 4 columns of train loss? And 3 of them seem to not converge. Can I locate my problem according to the output of the terminal?

**Comment by lululxvi** [comment]
See FAQ Q: What is the output of DeepXDE? How can I visualize the results?
